src/use_case/Adopt.java

```diff
@@ -26,6 +26,9 @@ public void execute(AdoptInputData adoptInputData) {
         uwu.markUnavailable();
         List<String> users = userDAO.removePetFromAllUserBookmarks(uwu.getPetID());
         petDAO.save(uwu);
+        for(String u : users){
+            u.addNotif("Pet " + uwu + " has been Adopted");
+        }
```

Comment:
for this part instead of printing it in the interactor can u make a presenter and do it there? check out use cases like get_breed and pet_bio and see how it's done there